arithmetization/src/main/java/net/consensys/linea/zktracer/module/mmu/MmuOperation.java

```diff
@@ -91,14 +89,18 @@ protected int computeLineCount() {
   }
 
   public int computeMmioLineCount() {
-    int sum = 0;
+    if (mmioLineCount != -1) {
+      return mmioLineCount;
+    }
+
+    mmioLineCount = 0;
```

Comment:
Was this method being called several times ? What explains this precaution ?

Reply:
Up to nb if tx in the batch, so ~200. We have the same mechanism for the hub line counting